design-system-docs/frontend/styles/_code.scss

```diff
@@ -26,6 +26,10 @@ pre code {
   border: $cads-border-width-small $cads-language__border-colour solid;
   padding: $cads-spacing-4;
 
+  &--with-button {
```

Comment:
Maybe we need to scope those to the component-example. When looking at the 'Using with Rails' sections, there is no margin above 'View Component Option' header that follows (https://630dc2463174f015e06e24be--ca-design-system.netlify.app/components/breadcrumbs/). E.g:
<img width="616" alt="image" src="https://user-images.githubusercontent.com/94383569/187489684-20416c43-a836-4902-8812-f61c6650db8c.png">

Reply:
Oh yeah, that's why I removed this originally 🙈. Let's try that!